packages/file_selector/file_selector_windows/lib/src/dart_file_selector_api.dart

```diff
@@ -0,0 +1,328 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ffi';
+
+import 'package:ffi/ffi.dart';
+import 'package:file_selector_windows/src/messages.g.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:win32/win32.dart';
+
+import 'dart_file_dialog.dart';
+import 'dart_file_open_dialog_api.dart';
+import 'dart_shell_item_api.dart';
+
+/// Dart native implementation of FileSelectorAPI
+class DartFileSelectorAPI extends FileDialog {
+  /// We need the file to exist. This value defaults to `false`.
```

Comment:
It doesn't look like this is the right comment for a constructor. 

Aside: Avoid we/you in docs and comments. Use the passive voice.
https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#use-the-passive-voice-recommend-do-not-require